.golangci.yml

```
@@ -49,7 +48,7 @@ linters-settings:
   errcheck:
     check-type-assertions: true
     check-blank: true
-    ignore: fmt:.*
+    exclude-functions: fmt:.*
   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)